bindings/ruby/extconf.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'mkmf'
+
+$LDFLAGS << ' -lc++'
```

Comment:
Using Apple's clang that comes with Xcode 7.3.1, the difference between clang++ and clang is that clang++ adds the following compiler flags:

```
-fdeprecated-macro
-fexceptions
-fcxx-exceptions
-stdlib=libc++
```

Enabling exceptions would make the binary bigger by some amount. I don't know if `-stdlib=libc++` has any effect on compilation, or if it's more for header search paths and linking.